test/iterator/basic_iterator.cpp

```diff
@@ -239,6 +239,20 @@ struct proxy_wrapper {
 		return *this;
 	}
 
+	proxy_wrapper const& operator=(const T& t) const
+	noexcept(std::is_nothrow_copy_assignable<T>::value)
+	{
+		get() = t;
+		return *this;
+	}
+
+	proxy_wrapper const& operator=(T&& t) const
+	noexcept(std::is_nothrow_move_assignable<T>::value)
+	{
+		get() = stl2::move(t);
+		return *this;
+	}
+
```

Comment:
File an issue for someone to look into it if/when we feel the need; it's not important enough to me to block the PR.